templates/bridgeserver2.yaml

```diff
@@ -15,6 +15,12 @@ Parameters:
   AwsVpcId:
     Type: "AWS::EC2::VPC::Id"
     Description: The AWS account's default VPC id
+  AwsVpcPrivateSubnets:
```

Comment:
AwsVpcId , AwsVpcPrivateSubnets and AwsVpcPublicSubnets can be referenced from the BridgeServer2-vpc template now so you probably don't need to pass it in.   For example, `us-east-1-BridgeServer2-vpc-PrivateSubnet`.  take a look at agora[1] as an example.

[1] https://github.com/Sage-Bionetworks/Agora-infra/blob/develop/templates/beanstalk.yaml#L263

Reply:
Changed